### DISCUSSION TITLE
[RFC] [Deprecation] -ms-high-constrast is in the process of being deprecated.

### DISCUSSION THREAD

**Discussion by daomtthuan** [storybookjs/storybook#28703] [RFC]
### Summary

-ms-high-constrast is in the process of being deprecated in the future.

### Problem Statement

-ms-high-constrast is in the process of being deprecated in the future. I see this warning on browser console. Is it ok for having a plan in next release of storybook?

![image](https://github.com/user-attachments/assets/7c61bd13-0aff-4cea-99c4-4c21ec477921)

### Implementation

https://blogs.windows.com/msedgedev/2024/04/29/deprecating-ms-high-contrast/

### Unresolved Questions

- [ ] Using a to do list makes it easy to resolve the questions as we move the RFC along.

**Comment by shilman** [comment]
This is a small change. Any chance you can submit a PR to remove it?

**Comment by Sidnioulz** [comment]
I can't find any reference to `-ms-high-contrast` in our codebase and have not seen this warning when testing Storybook in High Contrast Mode on Windows. Shall we close the RFC?
